src/uu/dd/src/dd.rs

```diff
@@ -116,6 +142,15 @@ impl Source {
                 Ok(m) => Ok(m),
                 Err(e) => Err(e),
             },
+            #[cfg(unix)]
+            Self::StdinFile(f) => match io::copy(&mut f.take(n), &mut io::sink()) {
```

Comment:
I suppose we can't use file seeking operations here, because stdin doesn't support that even when you open as file?

Reply:
I didn't try, but that was my assumption.

Reply:
Is it worth trying? I'm fine with the current implementation too.

Reply:
I can take a look now and remind myself what happens when using `seek()`. I'll report back

Reply:
It seems to cause a bunch of `Broken pipe (os error 32)` and `Illegal seek` errors throughout the tests if I use `f.seek()` here.

Reply:
Alright, thanks!